pyccel/codegen/printing/ccode.py

```diff
@@ -314,28 +314,6 @@ def __init__(self, filename, prefix_module = None):
         self._current_module = None
         self._in_header = False
 
-    def get_additional_imports(self):
```

Comment:
Did you leave `self._additional_imports` on purpose for the `CCodePrinter` class?

Reply:
Yes. It is initialised with a value so it is still needed. I can update the existing dict instead if you prefer?

Reply:
this is fine too! just wanted to make sure!